### ISSUE TITLE
Respect default argument of exception message in Exception inspection

### ISSUE THREAD

**Issue by afoeder** [kalessil/phpinspectionsea#402]
consider given
```
class AccessDeniedException extends \RuntimeException {
        public function __construct($message = 'Access Denied.')
```

At the moment
```
throw new AccessDeniedException();
```
gives inspection "..is throw without any message, please make a favor to yourself and add it".

![image](https://user-images.githubusercontent.com/1125168/28211789-bd70e5ba-689e-11e7-8f22-bad715ebc5b4.png)

![image](https://user-images.githubusercontent.com/1125168/28211803-c74e79c6-689e-11e7-9473-3bf405d669ed.png)

BTW: should read _is thrown_.

Thank you!

**Comment by kalessil** [comment]
Thanks for reporting @afoeder.
This probably related to #390, where reporting exception with no arguments should be in options.

**Comment by kalessil** [comment]
Right: it should be `throwN` =)

**Comment by kalessil** [comment]
Fixed in 7f125722a4c8d4605928e0541c972771ea2a4c5e
